src/Kunstmaan/ConfigBundle/Resources/config/routing.yml

```diff
@@ -0,0 +1,4 @@
+kunstmaanconfigbundle_default:
+    resource: "@KunstmaanConfigBundle/Controller/ConfigController.php"
```

Comment:
Better controller as service, so it is easy to override the controller class if needed